python/lsst/ctrl/bps/wms/panda/edgenode/cmd_line_decoder.py

```diff
@@ -128,4 +128,11 @@ def deliver_input_files(src_path, files, skip_copy):
     cmd_line = cmd_line.replace("{" + key + "}", value)
 
 print(cmd_line)
-sys.exit(os.WEXITSTATUS(os.system(cmd_line)))
+
+exit_status = os.system(cmd_line)
+exit_code = 1
```

Comment:
I don't know if using 1 as a default value will cause any confusion down the road with the exit 1 from the actual command.  I was wondering if could use 128 instead, but haven't tested it.

Reply:
AFAIK, exit code 1 is the "Catchall for general errors". It will be emitted if pipetask does not provide any exit code and there was not a signal which leads to the pipetask termination. If the program terminates beyond that two cases I would assign such a case to "undefined behavior" and classify as a general error. According to this: https://tldp.org/LDP/abs/html/exitcodes.html code 128 is the "Invalid argument to exit".